drivers/counter/Kconfig.imx_epit

```diff
@@ -0,0 +1,48 @@
+# Kconfig.imx_epit - IMX EPIT configuration options
+#
+# Copyright (c) 2018, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config TIMER_IMX_EPIT
+	bool "IMX EPIT driver"
+	depends on COUNTER && HAS_IMX_EPIT
+	help
+	  Enable the IMX EPIT driver.
+
+if TIMER_IMX_EPIT
+
+config TIMER_IMX_EPIT_1
+	bool "Timer 1"
+	help
+	  Enable Timer 1.
+
+config TIMER_IMX_EPIT_2
+	bool "Timer 2"
+	help
+	  Enable Timer 2.
+
+if TIMER_IMX_EPIT_1
+
+config TIMER_IMX_EPIT_1_PRESCALER
```

Comment:
Can we put the prescaler in DTS?

Reply:
I've seen it set in Kconfig [here](https://github.com/zephyrproject-rtos/zephyr/pull/8340/commits/fb5588d31e18e44841d288ddaa39f0886d1986cc) in #8340, so I wanted to make it the same.
If the most of counter/timer drivers use dts, I can change it.

Reply:
I think we should move the others to dts as well.

Reply:
Moved the prescaler setting into DTS for this pr.